src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,35 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
         return a / b;
     }
+
+    // Intentional errors added below
```

Comment:
5. **Error 5: Inconsistent naming convention**
   - The method `Subtract(int a, int b)` breaks the naming convention as method names should start with a lowercase letter. Maintain consistency in naming conventions by starting method names with a lowercase letter.